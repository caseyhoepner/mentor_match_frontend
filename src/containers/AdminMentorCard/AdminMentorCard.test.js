import React from 'react';
import { shallow } from 'enzyme';
import { AdminMentorCard } from './';

describe('AdminMentorCard', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<AdminMentorCard />)
  });

  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});