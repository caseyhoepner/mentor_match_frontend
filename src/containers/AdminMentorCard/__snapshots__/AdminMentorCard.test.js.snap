// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminMentorCard matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AdminMentorCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Name: Muthafucka
        </h1>,
        <h1>
          Stack Preference: Front-End
        </h1>,
        <h1>
          poop
        </h1>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Name: Muthafucka",
        },
        "ref": null,
        "rendered": "Name: Muthafucka",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Stack Preference: Front-End",
        },
        "ref": null,
        "rendered": "Stack Preference: Front-End",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "poop",
        },
        "ref": null,
        "rendered": "poop",
        "type": "h1",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Name: Muthafucka
          </h1>,
          <h1>
            Stack Preference: Front-End
          </h1>,
          <h1>
            poop
          </h1>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Name: Muthafucka",
          },
          "ref": null,
          "rendered": "Name: Muthafucka",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Stack Preference: Front-End",
          },
          "ref": null,
          "rendered": "Stack Preference: Front-End",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "poop",
          },
          "ref": null,
          "rendered": "poop",
          "type": "h1",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
